### PR TITLE
Touch up transport man page

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Touch up the containers-transports storage specificiation in the man page.

Fixes: #666

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by thmo** [comment]
That doesn't seem right, now it looks like this:
```
containers-storage{:storage-specifier} {image-id|docker-reference[@image-id]}
```
- the colon should be directly after `containers-storage`
- now there's a new space between the `}` and the `{` that shouldn't be there
- the original problem of the ticket is not finally solved, because in the text below, it still says
   ```
   The storage-specifier allows for referencing storage locations on the file system
   and has the format [[driver@]root[+run-root][:options]]
   ```
   where you cannot tell the difference between the outer `[` `]` (literal) and in the inner (option).

**Comment by TomSweeneyRedHat** [comment]
@thmo then I misunderstood what you were saying, and I'm still not clear what the final change should look like.  If you add it as a reply to this issue, I'll happily change it on the page.

File: docs/containers-transports.5.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ containers-transports - description of supported transports for copying and stor
 Tools which use the containers/image library, including skopeo(1), buildah(1), podman(1), all share a common syntax for referring to container images in various locations.
 The general form of the syntax is _transport:details_, where details are dependent on the specified transport, which are documented below.
 
-### **containers-storage:** [storage-specifier]{image-id|docker-reference[@image-id]}
+### **containers-storage**{:storage-specifier} {image-id|docker-reference[@image-id]}
```
Comment (mtrmac) [inline docs/containers-transports.5.md]: I think something like
```suggestion
### **containers-storage:**[[**storage-specifier**]**]{image-id|docker-reference[@image-id]}
```
would be accurate (I haven’t checked whether it renders correctly, neither in Markdown nor in the man page).

Comment (TomSweeneyRedHat) [reply docs/containers-transports.5.md]: This shows both brackets and everything highlighted.  I've pushed that out, please let me know if further adjustments are necessary.

**Comment by rhatdan** [comment]
LGTM

File: docs/containers-transports.5.md (hunk 1 above)
Comment (mtrmac) [reply docs/containers-transports.5.md]: Yeah, sadly the `###` format causes a `.SS` directive, which is bold by default. We could make the replacements italic, but a regular non-bold seems might not be possible to do with Markdown.

Oh well, I suppose we at least got rid of the space.

**Pull request merged**